File: Java/src/br/com/dominio/s08/Exe04b.java
package br.com.dominio.s08;

import java.util.Scanner;

public class Exe04b {

	public static void main(String[] args) {
		// variaveis
				int[] vetor1 = new int[20];
				int soma = 0;
				Scanner teclado = new Scanner(System.in);
				
				//entrada
				for (int i = 0; i < 20; i++) {
					System.out.printf("Informe o valor para o vetor %d/20\n", (i+1));
					vetor1[i] = teclado.nextInt();
					soma = (soma + vetor1[i]);
				}
				System.out.printf("A soma é %d", soma);
				
				teclado.close();

	}

}

File: Python/Seção08/Exe04b.py
#variaveis
vetor = []
#entradas
for n in range(0, 20):
    num = int(input("Informe {0}/20 valor para o vetor: ".format(n+1)))
    vetor.append(num)
print("A soma do vetor é {0}".format(sum(vetor)))